src/System.Net.Requests/src/System.Net.Requests.builds

```diff
@@ -8,10 +8,9 @@
     <Project Include="System.Net.Requests.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
-    <!-- Net46 facade is currently inbox for 4.0 
     <Project Include="System.Net.Requests.csproj">
-      <TargetGroup>net46</TargetGroup>
-    </Project> -->
+      <TargetGroup>net463</TargetGroup>
```

Comment:
Same comment as above. Why is this moving to net463?  It doesn't depend on new Desktop API surface.